mrp_operations_extension/models/mrp_production.py

```diff
@@ -119,6 +122,17 @@ def action_assign(self):
     def force_assign(self):
         self.move_lines.force_assign()
 
+    @api.multi
+    def write(self, vals, update=False):
+        if vals.get('date_planned', False):
```

Comment:
Lo de update viene de mrp_operations, es para que actualice la fecha de la OF, si no lo tiene marcado no lo hace. 
Lo de hacer lo mismo en el create no seria necesario, ya que en el create el workorder coge las fechas de las OFs. por lo que no habria que actualizar de nuevo la fecha de la OF.